components/dbutil/src/main/java/com/wso2telco/core/dbutils/DbUtils.java

```diff
@@ -389,7 +389,7 @@ public static String format(Double doubleData, int precision, int scale) throws
      *
      * @return the connection
      * @throws Exception the exception
-     */
+     *
     public Connection connect() throws Exception {
         System.out.println("-------- JDBC Connection Init ------------");
         Connection connection = null;
```

Comment:
If the methods are not being used, shouldn't those be removed from the code without keeping commented lines.

Reply:
removed.